gems/pending/MiqVm/MiqVm.rb

```diff
@@ -103,8 +103,10 @@ def openDisks(diskFiles)
 
       dInfo.hardwareId = dtag
       dInfo.baseOnly = @ost.openParent unless mode && mode["independent"]
-      dInfo.rawDisk = @ost.rawDisk # force raw disk for testing
+      dInfo.rawDisk = @ost.diskFormat == "raw" ? true : false
```

Comment:
Two things:
1. This line can just be: `dInfo.rawDisk = @ost.diskFormat == "raw"`
2. The diskFormat info you're passing in is OpenStack specific, so it shouldn't be passed to common code.
    It would probably be better to set `@ost.rawDisk` in the caller and leave this code unchanged.